src/applications/caregivers/config/form.js

```diff
@@ -108,6 +108,7 @@ const formConfig = {
     uuid,
     signature,
   },
+  dev: { disableWindowUnloadInCI: true },
```

Comment:
I added this new `dev` setting to prevent the window `beforeUnload` callback from opening an alert. I believe this is what was causing the Cypress timeouts after 40 minutes

<img width="975" alt="Leave site browser popup with cancel and leave buttons. Cypress is stuck in the background" src="https://github.com/user-attachments/assets/f631fcae-bd4e-4f52-9220-b2529919043f">

Cypress gets "stuck" and isn't able to proceed.